package.json

```diff
@@ -4,10 +4,12 @@
   "private": true,
   "dependencies": {
     "bulma": "^0.8.0",
+    "lint-fix": "^0.24.0",
     "mapbox-gl": "^1.5.1",
+    "prop-types": "^15.7.2",
```

Comment:
I am not sure if we are using prop-types in project but it is good to have it, let's keep it